cmd/snap/cmd_info.go

```
@@ -99,6 +106,15 @@ func tryDirect(w io.Writer, path string, verbose bool) bool {
 		return false
 	}
 
+	var sha3_384 string
+	if verbose && !osutil.IsDirectory(path) {
```

I would've just ignored the error, but this works as well :-)